concepticondata/conceptlists/Marrison-1967-909.tsv

```diff
@@ -0,0 +1,918 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	SIMILARITY
+Marrison-1967-1	1	among			
+Marrison-1967-2	2	and	1577	AND	2
+Marrison-1967-3	3	angry	1826	ANGRY	2
+Marrison-1967-4	4	animal	619	ANIMAL	2
+Marrison-1967-5	5	ankle	803	ANKLE	2
+Marrison-1967-6	6	answer	1275	ANSWER	2
+Marrison-1967-7	7	ant (black)			4
+Marrison-1967-8	8	ant (generic)	587	ANT	4
+Marrison-1967-9	9	ant (red)			4
+Marrison-1967-10	10	ant (white), 			4
+Marrison-1967-10	10	ant (white), termite	883	TERMITE	4
+Marrison-1967-11	11	arm	1673	ARM	2
+Marrison-1967-12	12	armlet			
+Marrison-1967-13	13	arrive	1387	ARRIVE	2
+Marrison-1967-14	14	arrow	977	ARROW	2
+Marrison-1967-15	15	ascend	705	GO UP (ASCEND)	2
+Marrison-1967-16	16	ashes	646	ASH	2
+Marrison-1967-17	17	ask (a question)	1309	ASK (INQUIRE)	4
+Marrison-1967-18	18	ask for	1828	ASK (REQUEST)	2
+Marrison-1967-19	19	at	1461	AT	2
+Marrison-1967-20	20	aunt	1272	AUNT	2
+Marrison-1967-21	21	awake / awaken	1602	WAKE UP	4
+Marrison-1967-22	22	axe	677	AXE	2
+Marrison-1967-23	23	baby	1289	BABY	2
+Marrison-1967-24	24	back (body)	1291	BACK	4
+Marrison-1967-25	25	bad	1292	BAD	2
+Marrison-1967-26	26	bag	2468	BAG	2
+Marrison-1967-27	27	bamboo	1927	BAMBOO	2
+Marrison-1967-28	28	banana	868	BANANA	2
+Marrison-1967-29	29	bank (edge)	113	SHORE	4
+Marrison-1967-30	30	bark (of tree)	1204	BARK	2
+Marrison-1967-31	31	basket (back, man's)			4
+Marrison-1967-32	32	basket (back, woman's)			4
+Marrison-1967-33	33	basket (generic)	1539	BASKET	4
+Marrison-1967-34	34	basket (large, storage)			4
+Marrison-1967-35	35	bat (animal)	1793	BAT	4
+Marrison-1967-36	36	be	1579	BE	2
+Marrison-1967-37	37	bead	1860	BEAD	2
+Marrison-1967-38	38	beam	1132	BEAM	2
+Marrison-1967-39	39	bean	832	BEAN	2
+Marrison-1967-40	40	bear (animal)	1820	BEAR	4
+Marrison-1967-41	41	bear (fruit)			4
```

Comment:
Somewhat late to the party, sorry if this has already been mentioned. There are a number these duplicates/triplicates/... in the list, probably a leftover of the automated mapping step.

Reply:
ok, working on it now.